bundle.yml

```diff
@@ -29,7 +29,7 @@ projects :
 
     - loki :
         git     : https://github.com/ecmwf-ifs/loki
-        version : main
```

Comment:
This needs flipping back once Loki PR is merged. I'll do this as part of the merge procedure.